src/test_record.py

```diff
@@ -0,0 +1,54 @@
+from microbit import *
+
+mouth_open = Image(
+    "09090:"
+    "00000:"
+    "09990:"
+    "90009:"
+    "09990"
+)
+mouth_closed = Image(
+    "09090:"
+    "00000:"
+    "00000:"
+    "99999:"
+    "00000"
+)
+play = Image(
+    "00000:"
+    "04740:"
+    "07970:"
+    "04740:"
+    "00000"
+)
+
+my_recording = audio.AudioRecording(duration=5000)
+my_track = None
+
+while True:
+    if button_a.is_pressed():
+        my_track = microphone.record_into(my_recording, wait=False)
+        display.show([mouth_open, mouth_closed], loop=True, wait=False, delay=150)
+        while button_a.is_pressed() and microphone.is_recording():
+            sleep(50)
+        microphone.stop_recording()
+        display.show(mouth_closed)
+        while button_a.is_pressed():
+            sleep(50)
+        display.clear()
+        my_track *= 2  # amplify volume
+    if button_b.is_pressed() and my_track:
```

Comment:
I think playing is triggered more reliably with was_pressed here, at least in the sim.

Reply:
Yes, good idea, now changed.